src/routes/resources/+page.svelte

```diff
@@ -237,11 +237,11 @@
                 <div
                     class="mx-4 mb-2 grid grid-cols-3 rounded-md rounded-t-none border-[1px] border-t-0 bg-base-200 p-2"
                 >
-                    <a
+                    <button
                         class="btn btn-outline self-center justify-self-start"
                         class:btn-disabled={$searchParams.page === 1}
-                        href={searchParams.calculateUrlWithGivenChanges({ page: $searchParams.page - 1 })}
-                        >{$translate('page.resources.table.navigation.previous.value')}</a
+                        on:click={() => ($searchParams.page -= 1)}
+                        >{$translate('page.resources.table.navigation.previous.value')}</button
```

Comment:
This used to be an `a` tag that helped the page feel snappier by doing the data prefetch. Since we removed the prefetch config, we can simplify things by making this a simple button that updates the searchParams store.